blob/lib/azure/storage/blob/blob_service.rb

```diff
@@ -137,6 +137,9 @@ def create_from_connection_string(connection_string, &block)
       # * +:default_endpoints_protocol+     - String. http or https
       # * +:use_path_style_uri+             - String. Whether use path style URI for specified endpoints
       # * +:ca_file+                        - String. File path of the CA file if having issue with SSL
+      # * +:ssl_version+                    - Symbol. The ssl version to be used
```

Comment:
>Symbol [](start = 46, length = 6)

Do you mean the symbols:  `:TLSv1_2`, `:TLSv1_3`, etc? It is better to define a type contains the available values, then everyone knows the valid parameter set.

Reply:
the values will change from time to time, define a type in the storage lib is not very good.
added the file reference link to the values.

Reply:
The link doesn't describe the parameter set clearly. Please add a sample in this comment.

---
In reply to: [223543476](https://github.com/Azure/azure-storage-ruby/pull/130#discussion_r223543476) [](ancestors = 223543476)

Reply:
added. thanks.